README.md

```diff
@@ -22,15 +22,15 @@ Let's examine what each piece of syntax does:
 
 *   **link**: the page is not active
 *   **current**: this is the current page
-*   **section**: the current page is a descendant of this page (i.e. on the URL `/about-us/company`, the "company" page is current, and the "about-us" page is "section."
+*   **section**: this is an _ancestor_ of the current page (i.e. if we are visiting `/about-us/company`, the "company" page is current, and the "about-us" page is "section."
```

Comment:
This line is missing the closing bracket

Reply:
Good spot!